generators/client/templates/angular/src/main/webapp/app/shared/auth/_principal.service.ts

```diff
@@ -9,8 +9,8 @@ import { <%=jhiPrefixCapitalized%>TrackerService } from '../tracker/tracker.serv
 @Injectable()
 export class Principal {
     private _identity: any;
-    private authenticated = false;
-    private authenticationState = new Subject<any>();
+    private _authenticated = false;
```

Comment:
Can we remove the underscores everywhere as it was originally to give as a feel as its a private var but now we have the private keyword so underscore is useless actually